src/panels/lovelace/ha-panel-lovelace.ts

```diff
@@ -135,21 +135,17 @@ class LovelacePanel extends hassLocalizeLitMixin(LitElement) {
         this._errorMsg = err.message;
         return;
       }
-      const {
-        generateLovelaceConfig,
-      } = await import("./common/generate-lovelace-config");
```

Comment:
decided to remove the dynamic import because it slows down for users in generated mode.